integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -285,19 +298,19 @@ protected String readGitCommitIdFromGitFile() {
     }
   }
 
-  private static boolean isGravitinoServerUp() {
-    JettyServerConfig jettyServerConfig =
-        JettyServerConfig.fromConfig(serverConfig, WEBSERVER_CONF_PREFIX);
-    String host = jettyServerConfig.getHost();
-    int port = jettyServerConfig.getHttpPort();
-    int timeout = 3000; // 3 second timeout
-
-    try (Socket socket = new Socket()) {
-      socket.connect(new java.net.InetSocketAddress(host, port), timeout);
-      LOG.info("Gravitino Server is up and running.");
-      return true;
+  public static boolean isGravitinoServerUp(String url) {
+    try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+      HttpGet request = new HttpGet(url);
+      String responseBody = httpClient.execute(request, new BasicHttpClientResponseHandler());
+      JsonObject jsonObject = new JsonParser().parse(responseBody).getAsJsonObject();
+      String version = jsonObject.get("version").getAsString();
```

Comment:
why check version here? is url must `VersionUrl`?

Reply:
No, I just want to confirm that the result is normal as the JSON content is expected in the following format:

```
{
	"version": "0.4.0"
     .....
}
```